x-pack/plugins/apm/ftr_e2e/cypress/plugins/index.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import Fs from 'fs';
+import { Client, HttpConnection } from '@elastic/elasticsearch';
+import { SynthtraceEsClient } from '@elastic/apm-synthtrace';
+import { createLogger, LogLevel } from '@elastic/apm-synthtrace';
+import { CA_CERT_PATH } from '@kbn/dev-utils';
+
+// ***********************************************************
+// This example plugins/index.ts can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+const plugin: Cypress.PluginConfig = (on, config) => {
+  // `on` is used to hook into various events Cypress emits
+  // `config` is the resolved Cypress config
+
+  const node = config.env.ES_NODE;
+  const requestTimeout = config.env.ES_REQUEST_TIMEOUT;
+  const isCloud = config.env.TEST_CLOUD;
+
+  let client: Client;
+
+  if (!isCloud) {
+    client = new Client({
+      node,
+      requestTimeout,
+      Connection: HttpConnection,
+    });
+  } else {
+    client = new Client({
+      tls: {
+        ca: Fs.readFileSync(CA_CERT_PATH, 'utf-8'),
+      },
+      node,
+      requestTimeout,
+      Connection: HttpConnection,
+    });
+  }
```

Comment:
nit: This is shorter and makes it clear that it's only the tsl settings that differ

```suggestion
const client = new Client({
  node,
  requestTimeout,
  Connection: HttpConnection,
  ...(isCloud ? { tls: { ca: Fs.readFileSync(CA_CERT_PATH, "utf-8") } } : {}),
});

```

Reply:
Cheers!